src/init.cpp

```diff
@@ -788,7 +788,9 @@ bool AppInit2(boost::thread_group& threadGroup)
     LogPrintf("Using at most %i connections (%i file descriptors available)\n", nMaxConnections, nFD);
     std::ostringstream strErrors;
 
-    LogPrintf("Using %u threads for script verification\n", nScriptCheckThreads);
+    // nScriptCheckThreads == 0 means we want to display it as 1 thread
+    unsigned int logScriptCheckThreads = (nScriptCheckThreads == 0) ? 1 : nScriptCheckThreads;
+    LogPrintf("Using %u threads for script verification\n", logScriptCheckThreads);
```

Comment:
The logic here is too complicated for it's own sake, but I think you're wrong here. 0 threads means 0 threads. No "script verification threads" are created.